### ISSUE TITLE
[Accessibility] [Progress Bar > Percentage] : Current value of progress bar is not announced when 'Increase' or 'Decrease' button is activated.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2101]
[A11Y-578](https://jira.cengage.com/browse/A11Y-578)

Environment:
Windows 11/Chrome 137/ JAWS 2025
Windows 11/Edge 136/ NVDA 2025
Mac 15/Safari/ VoiceOver

Steps to Reproduce:
1. Open the URL: https://react-magma.cengage.com/version/4.9.1/api/progress-bar/
2. Turn On screen reader (JAWS/NVDA).
2. Launch 'Progress Bar' component page from left navigation.
4. On the 'Progress Bar' page, under 'Percentage', navigate to 'Increase' or 'Decrease' button through Tab key and press Enter to change the progress bar value.
5. Verify that current value of progress bar is not announced.

Actual Result:
Current progress bar value is not announced when 'Increase' or 'Decrease' button is activated.

Refer to screencast:
https://app.screencast.com/Vj6b6yuttSxUQ

Expected Result:
When 'Increase' or 'Decrease' button is activated, the current progress bar value should be announced.

User Impact:
Screen reader users will not be able to identify the current value of progress bar and by what percentage the value is changed if it will not be announced to them.

WCAG Guideline:
4.1.3 Status Messages (AA)

Bug Fix Recommendation:
Add aria-live attribute to progress bar so that every time its value is changed, it could be announced by assistive technology.

**Not an issue with the component, but docs could be updated to showcase best practices.**
